catroid/src/main/java/org/catrobat/catroid/CatroidApplication.java

```diff
@@ -41,6 +41,7 @@
 import static org.catrobat.catroid.common.Constants.LANGUAGE_CODE;
 import static org.catrobat.catroid.common.Constants.LANGUAGE_TAG_KEY;
 
+@SuppressWarnings("PMD.AvoidUsingNativeCode")
```

Comment:
Instead of suppressing the warning for the whole class, just suppress the according methods (`loadSDKLib`, `loadNativeLibs`)